Cargo.toml

```diff
@@ -29,6 +29,7 @@ all-features = true
 default = ["bracketed-paste"]
 bracketed-paste = []
 event-stream = ["futures-core"]
+use-dev-tty = ["filedescriptor"]
```

Comment:
dont we want to make a freature flag: mio and refer to the mio dependency, and make this enabled by default?

Reply:
which would you like to be enabled by default?

 I think default features are a bit of a pain to deal with since you can't disable them individually, so if we keep mio as the default by making it a default feature it would be troublesome for users to opt out, as well as possibly breaking. this seemed cleaner since it makes the new behavior opt in and the feature can be deprecated easily later. Also it's intended to include #407.

Reply:
Good point. So then we temp can not opt out from mio but can opt in for the new feature.